pkg/webui/console/views/application-add/index.js

```diff
@@ -59,6 +69,7 @@ const validationSchema = Yup.object().shape({
   description: Yup.string(),
 })
 
+@withEnv
```

Comment:
we have env selectors. you can use `selectAsConfig` to pass `asEnabled` via `connect`